contrib/tools/python3/Lib/ssl.py

```diff
@@ -543,7 +543,10 @@ def load_default_certs(self, purpose=Purpose.SERVER_AUTH):
         if not isinstance(purpose, _ASN1Object):
             raise TypeError(purpose)
 
-        self.load_verify_locations(cadata=builtin_cadata())
+        cafile, capath, cadata = os.environ.get("SSL_CERT_FILE"), os.environ.get("SSL_CERT_DIR"), None
+        if cafile is None:
+            cadata=builtin_cadata()
+        self.load_verify_locations(cafile=cafile, capath=capath, cadata=cadata)
```

Comment:
This will break the old behavior:
* We always loaded builtin_cadata
* Right now, if there is cafile, then we don't upload it.

I suggest so:
```suggestion
        cafile, capath, cadata = os.environ.get("SSL_CERT_FILE"), os.environ.get("SSL_CERT_DIR"), builtin_cadata()
        self.load_verify_locations(cafile=cafile, capath=capath, cadata=cadata)
```
* Always loaded builtin_cadata
* If there is cafile, it will also be additionally uploaded.